README.md

```
@@ -1,61 +1,137 @@
-- [ ] Perform validations on the input to enable/disable the Next step button
 
 ### Milestone `Grande`
 
 - [ ] Create an OTP component
 - [ ] Upon successful email/name entry, get to OTP step
 
-## Milestone vocabulary
+## 🎯 Milestones
 
-Starbucks has the following denotations for sizes:
+### Tall
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)